src/views/conference/2020/index/index.jsx

```diff
@@ -91,6 +91,15 @@ const ConferenceSplash = () => (
                     <br />
                     <br />
                     <FormattedMessage id="conference-2020.registrationDelayed" />
+                    <br />
+                    <FormattedMessage 
+                        id="conference-2020.connectNow"
+                        values={{
+                           scratchInPracticeLink: <a href="https://sip.scratch.mit.edu/">
+                                <FormattedMessage id="conference-2020.scratchInPracticeText" />
+                            </a> 
```

Comment:
indent + trailing space
```suggestion
                           </a>
```